apps/src/templates/instructions/codeReviewV2/Comment.jsx

```diff
@@ -0,0 +1,297 @@
+import React, {Component} from 'react';
```

Comment:
This was copied from Comment component in code review V1, we have a separate ticket to update this component further